common/lib/capa/capa/responsetypes.py

```diff
@@ -248,7 +248,9 @@ def render_html(self, renderer, response_msg=''):
           - response_msg: a message displayed at the end of the Response
         """
         # render ourself as a <span> + our content
-        tree = etree.Element('span')
+        tree = etree.Element('section')
+        tree.set('class', 'wrapper-problem-response')
+        tree.set('tabindex', '-1')
```

Comment:
@cptvitamin Excellent explanation. If we go with "Display Name" then the wouldn't screen reader speak same thing for all the questions contained in the problem? I think "Question1", "Question 2" is a very good solution, I have talked with @muhammad-ammar as well on this and we think it's very much do able, since `section` elements are rendered by responsetypes so it doesn't matter if they are seperated with dashes or not.